optaplanner-core/src/main/java/org/optaplanner/core/api/score/stream/bi/BiConstraintStream.java

```diff
@@ -198,6 +198,166 @@
         return join(otherClass, AbstractTriJoiner.merge(joiners));
     }
 
+    // ************************************************************************
+    // If (Not) Exists
```

Comment:
style: Not and Exists without capital. only first word has capital in comments

Reply:
BTW, in general for titles (including docs), we're now following scientific titles, not newspaper titles. So basically only the first word has a capital.